src/plugins/home/public/application/components/add_data/add_data.tsx

```diff
@@ -66,19 +67,38 @@ export const AddData: FC<Props> = ({ addBasePath, application, isDarkMode }) =>
 
             <EuiSpacer />
 
-            <EuiFlexGroup gutterSize="m" responsive={false} wrap>
+            <EuiFlexGroup gutterSize="m">
+              {isCloudEnabled && (
+                <EuiFlexItem grow={false}>
+                  {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
+                  <EuiButton
+                    data-test-subj="guidedOnboardingLink"
+                    fill
+                    href={addBasePath('#/getting_started')}
+                    onClick={(event: MouseEvent) => {
+                      trackUiMetric(METRIC_TYPE.CLICK, 'guided_onboarding_link');
+                    }}
+                  >
+                    <FormattedMessage
+                      id="home.addData.guidedOnboardingLinkLabel"
+                      defaultMessage="Launch Guided setup"
+                    />
+                  </EuiButton>
+                </EuiFlexItem>
+              )}
               <EuiFlexItem grow={false}>
                 <RedirectAppLinks application={application}>
                   {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
                   <EuiButton
                     data-test-subj="homeAddData"
-                    fill
+                    fill={!isCloudEnabled}
```

Comment:
[nit] Might be helpful to add a comment here to explain why the `fill` value changes on cloud.